view/adminhtml/web/template/ui/grid/filters/elements/ui-select.html

```diff
@@ -0,0 +1,226 @@
+                <div class="action-menu-item"
+                     data-bind="
+                        css: {
+                            _selected: $parent.isSelectedValue(option),
+                            _hover: $parent.isHovered(option, $element),
+                            _expended: $parent.getLevelVisibility($data) && $parent.showLevels($data),
+                            _unclickable: $parent.isLabelDecoration($data),
+                            _last: $parent.addLastElement($data),
+                            '_with-checkbox': $parent.showCheckbox
+                        },
+                        click: function(data, event){
+                            $parent.toggleOptionSelected($data, $index(), event);
+                        },
+                        clickBubble: false
+                ">
+                    <!-- ko if: $data.optgroup && $parent.showOpenLevelsActionIcon-->
+                    <div class="admin__action-multiselect-dropdown"
+                         data-bind="
+                            click: function(event){
+                                $parent.showLevels($data);
+                                $parent.openChildLevel($data, $element, event);
+                            },
+                            clickBubble: false
+                         ">
+                    </div>
+                    <!-- /ko-->
+                    <!--ko if: $parent.showCheckbox-->
+                    <input
+                            class="admin__control-checkbox"
+                            type="checkbox"
+                            tabindex="-1"
+                            data-bind="attr: { 'checked': $parent.isSelected(option.value) }">
+                    <!-- /ko-->
+                    <label class="admin__action-multiselect-label">
+                        <span data-bind="text: option.label"></span>
+                        <span
+                                if="$parent.getPath(option)"
+                                class="admin__action-multiselect-item-path"
+                                data-bind="text: $parent.getPath(option)"></span>
+                    </label>
+                </div>
+                <!-- ko if: $data.optgroup -->
+                <!-- ko template: {name: $parent.optgroupTmpl, data: {root: $parent, current: $data}} -->
+                <!-- /ko -->
+                <!-- /ko-->
+            </li>
+            <!-- /ko -->
+        </ul>
+        <!-- ko if: $data.closeBtn -->
+        <div class="admin__action-multiselect-actions-wrap">
+            <button class="action-default"
+                    data-action="close-advanced-select"
+                    type="button"
+                    data-bind="click: outerClick">
+                <span translate="closeBtnLabel"></span>
+            </button>
+        </div>
+        <!-- /ko -->
+    </div>
+</div>
+
+<button class="admin__field-fallback-reset" type="button" data-bind="
```

Comment:
Hier voeg jij een knop toe aan het template, moeten er dan geen BEGIN/END tags bij komen?

Reply:
Geen override, het is nu onze template!

Reply:
In feite is het een kopie van een Magento template. Denk jij niet dat het nuttig is om ook bij kopiëen van templates BEGIN/END tags toe te passen?

Dan kunnen we rekening houden met onze changes als we de modules weer eens gaan bekijken.